field_char_transformed/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================================
+Transform data on character fields
+==================================
+
+This module was written to give programmers a convenient way to transform text input before it is written into the database or before it is read from it.
+
+As a practical example, FieldCharStripped is provided, which strips whitespace from the value before it is committed.
+
+Configuration
+=============
+
+This module is a dependency to be used by programmers, not end users. There's no configuration to be done via the UI.
+
+Usage
+=====
+
+Depend on this module and import the field classes you need::
+
+    from openerp.addons.field_char_transformed import FieldCharTransformed, FieldCharStripped
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/server-tools/issues/new?body=module:%20field_char_transformed%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
Hi @hbrunn, thanks for the contribution!
Can you please use the latest README template and remove this last line?